app/helpers/proposals_helper.rb

```diff
@@ -11,4 +11,13 @@ def datespan_header(start_date, end_date)
       end
     end
   end
+
+  def truncate_link(link)
+    if (link.size > 70)
+      truncated_link = link[0...69] << "..."
+      return truncated_link
+    else
+      return link
+    end
+  end
```

Comment:
Rails actually has a built-in [`truncate`](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-truncate) method...does the same thing, but mind using that instead?